share/functions/__fish_svn_prompt.fish

```diff
@@ -0,0 +1,169 @@
+# colour of the revision number to display in the prompt
+set -g __fish_svn_prompt_color_revision yellow
+
+# setting the prompt status separator character
+set -g __fish_svn_prompt_char_separator "|"
+
+# ==============================
+
+# SVN status symbols
+# Do not change these! These are the expected characters that are output from `svn status`, they are taken from `svn help status`
```

Comment:
Since this gets called 7x in a pretty tight loop i was worried that a local variable setup/teardown would cause a bit of unwanted overhead to an already expensive function call. If you think that is fine, I can make that change but I am concerned about the performance of doing this.